src/components/Book.js

```diff
@@ -9,25 +9,63 @@ const Book = (props) => {
   const dispatch = useDispatch();
 
   const deleteBook = (event) => {
-    const { id } = event.target.parentElement;
+    const { id } = event.target.parentElement.parentElement.parentElement;
     removeBookAPI(id, dispatch);
   };
 
   return (
-    <li id={id}>
-      <h3>
-        {title}
-        {' '}
-        {author}
-      </h3>
-      <button
-        type="button"
-        onClick={(event) => {
-          deleteBook(event);
-        }}
-      >
-        remove
-      </button>
+    <li className="book-panel" id={id}>
+      <div className="book-specs">
+        <h3 className="category">Action</h3>
+        <h2 className="title">{title}</h2>
+        <h6 className="author">{author}</h6>
+        <div className="buttonsWrapper">
+          <button className="button" type="button">
+            Comments
+          </button>
+          <div className="separator" />
+          <button
+            className="button"
+            type="button"
+            onClick={(event) => {
+              deleteBook(event);
+            }}
+          >
+            remove
+          </button>
+          <div className="separator" />
+          <button className="button" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="panel-middle">
+        <div className="book-progress">
+          <div className="circle-wrap">
+            <div className="circle">
+              <div className="mask full">
+                <div className="fill" />
+              </div>
+              <div className="mask half">
+                <div className="fill" />
+              </div>
+              <div className="inside-circle"> 75% </div>
+            </div>
+          </div>
+          <div>
+            <h5 className="progress-percentage">75%</h5>
+            <h6 className="progress-completed">Completed</h6>
```

Comment:
- [ ] Let's remove the 75% from the inside of the progress bar. It's important that if we're working for a client we understand that the freedom we will have to change the requested design will vary from project to project. This time we're working for a very strict client. :sweat_smile: 

Yours:
![image](https://user-images.githubusercontent.com/79114233/141554845-b8a9b536-d82b-4f6e-aa0a-fa2b3bca3c7c.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/141555032-88222e22-6baf-4f3c-a348-6f949b1a0723.png)

Reply:
Ok, thanks for mentioning that